packages/components-react/src/link/Link.css

```
@@ -0,0 +1,43 @@
+.lux-link {
+  display: inline-flex;
```

```suggestion
  display: inline flex;
```

Deze nieuwe syntax is de new 'n shiny en daarbij de nieuwe standaard. Het is baseline dus we kunnen het veilig gebruiken.

https://developer.mozilla.org/en-US/docs/Web/CSS/display